src/instruments/mettler_toledo/mt_sics.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
+"""
+Provides support for the Mettler Toledo balances via Standard Interface Command Set.
+"""
+
+import warnings
+
+from instruments.abstract_instruments import Instrument
+from instruments.units import ureg as u
+from instruments.util_fns import assume_units
+
+
+class MTSICS(Instrument):
+    """
+    Instrument class to communicate with Mettler Toledo balances using the MT-SICS
+    Standared Interface Command Set.
+
+    Example usage:
+
+    >>> import instruments as ik
+    >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+    >>> inst.weight_stable
+    <Quantity(120.2, 'gram')>
+    """
+
+    def __init__(self, filelike):
+        super().__init__(filelike)
+        self.terminator = "\r\n"
+
+    def clear_tare(self):
+        """
+        Clear the tare value.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.clear_tare()
+        """
+        _ = self.query("TAC")
+
+    def reset(self):
+        """
+        Reset the balance.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.reset()
+        """
+        _ = self.query("@")
+
+    def tare(self):
+        """
+        Tare the balance after stable weight is obtained.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.tare()
+        """
+        _ = self.query("T")
+
+    def tare_immediately(self):
+        """
+        Tare the balance immediately.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.tare_immediately()
+        """
+        _ = self.query("TI")
+
+    def zero(self):
+        """
+        Zero the balance after stable weight is obtained.
+
+        Terminates processes such as zero, tare, calibration and testing etc.
+        If the device is in standby mode, it is turned on. This function sets the
+        currently read and the tare value to zero.
+
+        Example usage:
+
+        >>> import instruments as ik
+        >>> inst = ik.mettler_toledo.MTSICS.open_serial('/dev/ttyUSB0', 9600)
+        >>> inst.zero()
+        """
+        _ = self.query("Z")
+
+    def zero_immediately(self):
```

Comment:
My initial thought here is that this should be part of `zero`:

`def zero(immediately=False):`

Reply:
or something equivalent for the kwarg

Reply:
good point, i like that idea! way more concise and i guess most users don't want to use the `immediately` capability anyway

Reply:
this one is done, the same with `tare_immediately()` above.